it.stethewwolf.gresistor.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app-id": "it.stethewwolf.gresistor",
```

Comment:
`stethewwolf.it` doesn't look like a valid (registered) domain which mean the app-id is not conformant to the requirements.

Reply:
seems like it is a leftover there are two manifest.